### PR TITLE
WRQ-5672: Support m3u8 video format for enact video player

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
As a supplement for https://github.com/enactjs/samples/pull/216 PR, 
_sandstone/pattern-video-player-custom/README.md_ has to be updated.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
In README.md file, I added more explanations of how to support the m3u8 format with enact VideoPlayer while developing the app.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-5672

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim ([jiye.kim@lge.com](mailto:jiye.kim@lge.com))

### CODE REVIEW

File: sandstone/pattern-video-player-custom/README.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ Run `npm install` then `npm run serve` to have the app running on [http://localh
 
 Videos in [videos.js](src/App/videos.js) are added to VideoPlayer.
 
-If the video format is `m3u8`, you can play it by writing the mime type of the video as `application/x-mpegURL` in [videos.js](src/App/videos.js).
+If the video format is `m3u8`, you can play it by writing url of the video and mime type as `application/x-mpegURL` in [videos.js](src/App/videos.js).
+We support `m3u8` format using [hls.js](https://github.com/video-dev/hls.js) which is imported in [App.js](src/App/App.js).
```
Comment (juwonjeong) [inline sandstone/pattern-video-player-custom/README.md]:
```suggestion
To use `m3u8` format with enact VideoPlayer in your app, you need to import and use [hls.js](https://github.com/video-dev/hls.js) as in [App.js](src/App/App.js).
```

File: sandstone/pattern-video-player-custom/README.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ Run `npm install` then `npm run serve` to have the app running on [http://localh
 
 Videos in [videos.js](src/App/videos.js) are added to VideoPlayer.
 
-If the video format is `m3u8`, you can play it by writing the mime type of the video as `application/x-mpegURL` in [videos.js](src/App/videos.js).
+If the video format is `m3u8`, you can play it by writing url of the video and mime type as `application/x-mpegURL` in [videos.js](src/App/videos.js).
+We support `m3u8` format using [hls.js](https://github.com/video-dev/hls.js) which is imported in [App.js](src/App/App.js).
+[App.js](src/App/App.js) brings video information from [videos.js](src/App/videos.js) and bind source and video element in video player depending on mime type.
```
Comment (juwonjeong) [inline sandstone/pattern-video-player-custom/README.md]:
```suggestion
In this sample app, [App.js](src/App/App.js) brings video information from [videos.js](src/App/videos.js) and bind source and video element in video player depending on mime type.

```

**Comment by juwonjeong** [comment]
I think it would be better to update the resolution of the PR description 
`In README.md file, I added more explanations of how to support the `m3u8` format with enact VideoPlayer while developing the app.`

File: sandstone/pattern-video-player-custom/README.md (hunk 1 above)
Comment (vJIYEv) [reply sandstone/pattern-video-player-custom/README.md]: I accepted your suggestion.

File: sandstone/pattern-video-player-custom/README.md (hunk 2 above)
Comment (vJIYEv) [reply sandstone/pattern-video-player-custom/README.md]: I accepted your suggestion.

**Comment by seunghoh** [comment]
Update README in the https://github.com/enactjs/samples/pull/222

**Pull request closed without merging**